### ISSUE TITLE
is there a way to remove background from the exported splat (from splatfacto) ? 

### ISSUE THREAD

**Issue by brahimfarhat** [nerfstudio-project/nerfstudio#3158]
is there a way to remove background from the exported splat (from splatfacto) ? 

I used ns-export with the following options but it crashed 

ns-export gaussian-splat --load-config outputs/object/splatfacto/2024-05-22_170348/config.yml --output-dir ../app/ --obb-center -0.0700000003 -0.0299999993 -0.0799999982 --obb_scale 1.0000000000 1.0000000000 1.0000000000 --obb_rotation 0.0000000000 0.0000000000 -0.7805307416_

[14:46:48] Auto image downscale factor of 4                                                 nerfstudio_dataparser.py:484
Loading latest checkpoint from load_dir
✅ Done loading checkpoint from outputs/object/splatfacto/2024-05-22_170348/nerfstudio_models/step-000019999.ckpt
model/nerfstudio/nerfstudio/data/scene_box.py:105: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  comp_l = torch.tensor(-S / 2)
model/nerfstudio/nerfstudio/data/scene_box.py:106: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  comp_m = torch.tensor(S / 2)

Any help ?
